lib/web/mage/menu.js

```diff
@@ -19,8 +19,7 @@ define([
         options: {
             responsive: false,
             expanded: false,
-            showDelay: 42,
-            hideDelay: 300,
+            delay: 300,
```

Comment:
Original options that were removed still used on 104 and 108 lines in this file, so after removal them from options - they will be not defined. It looks like you've added regression. Could you check it?